### PR TITLE
option to disable compression for range request's response

### PR DESCRIPTION
**Pull request by vmamidi**
Provide an option to disable compression for a request with Range header.

### CODE REVIEW

File: doc/admin-guide/plugins/compress.en.rst [hunk 1]
```diff
@@ -103,6 +103,12 @@ versions of the content as :term:`alternates <alternate>`. When set to
 ``false``, |TS| will cache only the compressed or decompressed variant returned
 by the origin. Enabled by default.
 
+range-request
+-----
+
+When set to ``true``, causes |TS| to compress responses to Range Requests.
+Disabled by default.
```
Comment (mlibbey) [inline doc/admin-guide/plugins/compress.en.rst:110]: Lets add:
WARNING! Do NOT set this to ``true`` if cache is set to ``false``. This combination will deliver corrupt content.

**Comment by randall** [comment]
I'm just assuming this is too late for 9.0.

**Review by ywkaras — APPROVED** [review]

**Comment by zwoop** [comment]
Cherry-picked to v9.0.x branch.

**Comment by ezelkow1** [comment]
Can we get a backport PR for 8.1.x for this?

**Pull request merged**